catalog/additionalInfo.md

```diff
@@ -0,0 +1,3 @@
+## Additional Info
+
+Some additional info could be said here.
```

Comment:
@danielgolden Do we want an additionalInfo.md with just placeholder text? Or remove it all together?

Reply:
Good point, will remove it and then merge it.